docs/tutorials/articles/sales_dashboard/step_03/step_03.md

```diff
@@ -0,0 +1,73 @@
+---
+hide:
+  - toc
+---
+
+The full code for this step is available 
```

Comment:
We should have the normal download button. Check what is done right now for this tutorial